src/jobs/index.js

```diff
@@ -23,9 +24,23 @@ const updateEmailServices = () => {
   cron.schedule('0 1 * * *', () => UserController.updateEmailServices())
 }
 
+const updateMonthlyRanking = () => {
+  cron.schedule('00,20,40 * * * *', () =>
```

Comment:
Coloquei de 20 em 20 min intercalando com uma diferença de 10 min entre os dois jobs. A gente combinou a cada uma hora, mas acho assim o ranking fica mais atualizado sem forçar demais. 

Qualquer coisa posso deixar de hora em hora mesmo.